src/components/vectorfeedback/partials/vectorfeedback.html

```diff
@@ -7,13 +7,9 @@
         <p class="ga-vector-feedback-text" translate>vector_feedback_description_text</p>
         <hr>
         <div class="ga-vector-feedback-item">
-            <h4 class="ga-vector-feedback-item-title pull-left" translate>vector_feedback_select_style</h4>
-            <div class="ga-vector-feedback-item-select-wrapper">
-                <select class="form-control">
-                    <option>dummy</option>
-                </select>
-            </div>
+            <button class="btn btn-default" ng-click="applyNextStyle()" translate>vector_feedback_select_style</button>
         </div>
+        <hr>
```

Comment:
I think we should have the translated style id somewhere (day, night, fjord etc..). I know we don't have this information.
But instead of stylesUrls, we could have
```javascript
glStyles = [
{
id:
url:
edits: []
},
{
}
]
```
This would also make it easier to `apply the next style` and allow for different layers across the layers.

What do you think?